### ISSUE TITLE
FatalErrorException in Node.php line 77

### ISSUE THREAD

**Issue by adielhercules** [pagekit/pagekit#502]
I am having this issue with pagekit fresh installation:

FatalErrorException in Node.php line 77:
Compile Error: Trait method find has not been applied, because there are collisions with other trait methods on Pagekit\Site\Model\Node

I am using the latest version of pagekit, php 5.4.12 , sqlite 3 (also tryed with mysql driver but doesnt succeed on installation).

**Comment by janschoenherr** [comment]
There is only one _find_ method there. Could you setup a clean installation and try with that?

**Comment by adielhercules** [comment]
Weird, I tried several times and I got same result, the installation with sqlite is going perfect (mysql doesnt work, it dies in a 500 response when it does a XMLHTTPRequest while installation is being performed) but when the setup redirects the fatal error is coming, I tryed to open /admin and / urls but the error is on there, I have debug mode enabled.

I inspected the code and yes, the NodeModelTrait only has one find method but the error is still there. Any suggestion?
